src/main/java/nl/b3p/geotools/data/linker/blocks/Action.java

```diff
@@ -86,7 +86,7 @@ protected Integer getAttributeId(EasyFeature feature, String columnName) throws
      * @return the clean String
      */
     public static String fixTypename(String in) {
-        String allowed = "qwertyuiopasdfghjklzxcvbnm1234567890_";
+        String allowed = "qwertyuiopasdfghjklzxcvbnm1234567890_:";
```

Comment:
dit zet de deur open voor problemen met databases; de `:` is waarschijnlijk alleen maar geldig voor WFS en niet voor RDBMS. Hier moet een check voor WFS-T en dan de `:` toestaan.